### PR TITLE
Auto Translations 🌎

### PR DESCRIPTION
**Pull request by applenick**
Experimental chat translations!

### CODE REVIEW

File: src/main/java/dev/pgm/community/translations/TranslationConfig.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package dev.pgm.community.translations;
+
+import dev.pgm.community.feature.config.FeatureConfigImpl;
+import java.util.List;
+import org.bukkit.configuration.Configuration;
+
+public class TranslationConfig extends FeatureConfigImpl {
+
+  private static final String KEY = "translations";
```
Comment (Electroid) [inline src/main/java/dev/pgm/community/translations/TranslationConfig.java:10]: Let's add an API Key field, as I'm going to add it very soon: `translations.api-key`

Comment (Electroid) [reply src/main/java/dev/pgm/community/translations/TranslationConfig.java:10]: If provided, it should add an `Authorization: Bearer {api-key}` header to each request.

File: src/main/java/dev/pgm/community/utils/WebUtils.java [hunk 2]
```diff
@@ -10,20 +10,75 @@
...
 public class WebUtils {
 
-  private static final String RANDOM_NAME_API = "https://api.gamertag.dev/random"; // :)
-  private static final String USERNAME_API = "https://api.ashcon.app/mojang/v2/user/";;
+  // A big thanks to @Electroid for all these awesome APIs :)
+  private static final String RANDOM_NAME_API = "https://api.gamertag.dev/random";
+  private static final String USERNAME_API = "https://api.ashcon.app/mojang/v2/user/";
+  private static final String TRANSLATE_API = "https://api.gamertag.dev/translate";
+
+  public static CompletableFuture<Translation> getTranslated(
```
Comment (Electroid) [inline src/main/java/dev/pgm/community/utils/WebUtils.java:35]: 1. Let's use a `Cache` for the last 100 messages.
2. Let's only send `Accept-Language` for locales that are online.

File: src/main/java/dev/pgm/community/translations/TranslationConfig.java (hunk 1 above)
Comment (applenick) [reply src/main/java/dev/pgm/community/translations/TranslationConfig.java:10]: Great idea! Added 👍

**Pull request merged**